tests/components/devolo_home_network/test_config_flow.py

```diff
@@ -171,6 +173,45 @@ async def test_abort_if_configued(hass: HomeAssistant):
     assert result3["reason"] == "already_configured"
 
 
+@pytest.mark.usefixtures("mock_device")
+@pytest.mark.usefixtures("mock_zeroconf")
+async def test_form_reauth(hass: HomeAssistant):
```

Comment:
This test shows that the reauth flow works as expected.
What triggers a reauth flow though?

Reply:
Nothing yet, because the devices have API endpoints that work without giving the password. Currently only those endpoints are used. But I plan to a add those endpoints next, that require a password. Unfortunately, the device does not have a way to check, that the password is correct before trying to use one of those endpoints.

So basically in future, there will e.g. be a switch platform that can turn on/off the LED of the device. Getting the state of the LED will always work, but setting the LED will trigger the reauth flow if the device is password protected.

Reply:
No problem, I have a platform using it more or less ready, So it can wait a bit longer.